public/xash3d_mathlib.h

```diff
@@ -77,6 +77,9 @@ GNU General Public License for more details.
 #define Q_round( x, y )	(floor( x / y + 0.5f ) * y )
 #define Q_rint(x)		((x) < 0.0f ? ((int)((x)-0.5f)) : ((int)((x)+0.5f)))
 
+#ifdef XASH_IRIX
+#undef isnan
```

Comment:
Is there something wrong about `isnan` on IRIX?

Reply:
The default system `isnan` seems to be broken in some cases. In one specific instance it causes the build to fail. The GCC 9.2 toolchain for IRIX is supposed to prefer a fixed version but for some reason this seems to not work on my Octane2 running IRIX 6.5.30 but did work on my other machine running 6.5.22. However I haven't been able to figure out the problem yet so this serves as fix for the moment.